ArcPicker/ArcPicker.js

```diff
@@ -185,8 +193,10 @@ const ArcPickerBase = kind({
 	computed: {
 		arcSegments: (props, context) => {
 			const {accent: accentColor} = context || {};
-			const {children, endAngle, isFocused, onClick, radius, selectionType, skinVariants, startAngle, strokeWidth, value} = props;
-			const backgroundColor = props.backgroundColor || (skinVariants && skinVariants.night ? '#444444' : '#888888');
+			const {children, endAngle, isFocused, onClick, radius, selectionType, skin, skinVariants, startAngle, strokeWidth, value} = props;
+
+			const backgroundColorNight = skin === 'titanium' ? '#ffffff' : '#888888';
```

Comment:
Hm.. backgoundColorNight would make confused with skinVariants.night. Do we also need to check specific skin here? Is there any way not to use hard coded color values here?